app/controllers/users.js

```diff
@@ -153,7 +156,36 @@ export function create(req, res) {
     });
   }
 }
-
+export const logIn = (req, res) => {
+  const { email, password } = req.body;
+  if (password && email) {
+    if (validator.isEmail(req.body.email)) {
+      User.findOne({
+        email
+      }).then((existingUser) => {
+        if (!existingUser) {
+          return res.status(404).send({ message: 'You seem to have not registered this account with us' });
```

Comment:
Great work here, A 404 definitely does the job. However, since we're working with authentication, do you think a 401 status code will be better suited here.